bin/integrationTest/spring/resources.groovy

```diff
@@ -1,7 +1,14 @@
+<<<<<<< HEAD
 import groovyshool.auth.UserPasswordEncoderListener
 import groovyschool.auth.UserPasswordEncoderListener
 // Place your Spring DSL code here
 beans = {
     userPasswordEncoderListener(UserPasswordEncoderListener)
     userPasswordEncoderListener(UserPasswordEncoderListener)
 }
+||||||| merged common ancestors
+=======
+// Place your Spring DSL code here
+beans = {
+}
+>>>>>>> pull from upstream
```

Comment:
This is a merge conflict that needs to be resolved.
take a look at https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts